### ISSUE TITLE
Single arguments should be placed on same line as invocation

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#528]
We should issue a warning in case an argument of an invocation is on a different line than the invocation itself.

Following example should issue a warning:
```c#
DoSomething(
  123);
```

It should be fixed to:
```c#
DoSomething(123);
```
